README.md

```diff
@@ -13,7 +11,7 @@ This is my first portfolio project, that is responsive to all screensizes. Mobil
 
 ## Live Demo
 
-[link will be updated as soon as the project is complete]
+[click here for live demo] (https://favourezeugwa.github.io/First-Portfolio/)
```

Comment:
- [ ] [Optional] Kindly remove the space between braces to make it work like that [click here for live demo](https://favourezeugwa.github.io/First-Portfolio/).